apps/condo/domains/meter/components/CreateMeterReadingsActionBar.tsx

```diff
@@ -1,39 +1,36 @@
 import { Form } from 'antd'
 import { isEmpty } from 'lodash'
-import get from 'lodash/get'
 import React from 'react'
 
-import { PlusCircle } from '@open-condo/icons'
 import { useIntl } from '@open-condo/next/intl'
-import { useOrganization } from '@open-condo/next/organization'
 import { ActionBar, Tour } from '@open-condo/ui'
 
 import { ButtonWithDisabledTooltip } from '@condo/domains/common/components/ButtonWithDisabledTooltip'
-import { METER_PAGE_TYPES } from '@condo/domains/meter/utils/clientSchema'
+import { MeterPageTypes, METER_TAB_TYPES } from '@condo/domains/meter/utils/clientSchema'
 
 
 const PROPERTY_DEPENDENCY = ['property']
 const handleShouldUpdate = (prev, next) => prev.unitName !== next.unitName
+type CREATE_METER_READINGS_ACTION_BAR_PROPS = {
+    handleSave: () => void
+    isLoading: boolean,
+    newMeterReadings: Array<any>
```

Comment:
Why any? Is there any option to use generated type or specify basic object definition? If there is no option for that you can use `unknown` keyword